tools/viewer/src/viewer.js

```diff
@@ -340,42 +340,39 @@ Object.assign(Viewer.prototype, {
         var cubemap = new pc.Asset('helipad', 'cubemap', {
             url: assetsFolder + "/cubemaps/Helipad.dds"
         }, {
-            textures: [
-                assetsFolder + "/cubemaps/Helipad_posx.png",
-                assetsFolder + "/cubemaps/Helipad_negx.png",
-                assetsFolder + "/cubemaps/Helipad_posy.png",
-                assetsFolder + "/cubemaps/Helipad_negy.png",
-                assetsFolder + "/cubemaps/Helipad_posz.png",
-                assetsFolder + "/cubemaps/Helipad_negz.png"
-            ],
+            // textures: [
+            //     assetsFolder + "/cubemaps/Helipad_posx.png",
+            //     assetsFolder + "/cubemaps/Helipad_negx.png",
+            //     assetsFolder + "/cubemaps/Helipad_posy.png",
+            //     assetsFolder + "/cubemaps/Helipad_negy.png",
+            //     assetsFolder + "/cubemaps/Helipad_posz.png",
+            //     assetsFolder + "/cubemaps/Helipad_negz.png"
+            // ],
             magFilter: pc.FILTER_LINEAR,
             minFilter: pc.FILTER_LINEAR_MIPMAP_LINEAR,
             anisotropy: 1,
             type: pc.TEXTURETYPE_RGBM
         });
-        cubemap.loadFaces = true;
+        // cubemap.loadFaces = true;
```

Comment:
Delete?

Reply:
I left these as a reminder to myself and others what's needed to load faces. I've removed though.